example/timer/odp_timer_accuracy.c

```diff
@@ -446,23 +538,37 @@ static int start_timers(test_global_t *test_global)
 	test_global->start_ns = start_ns;
 	test_global->period_tick = odp_timer_ns_to_tick(timer_pool, period_ns);
 
-	/* When mode is 1, set only one burst of timers initially */
-	if (mode)
+	/* When mode is not one-shot, set only one burst of timers initially */
+	if (mode != MODE_ONESHOT)
 		num_tmo = 1;
 
 	for (i = 0; i < num_tmo; i++) {
 		for (j = 0; j < burst; j++) {
 			timer_ctx_t *ctx = &test_global->timer_ctx[idx];
 			odp_timer_start_t start_param;
 
-			nsec = offset_ns + (i * period_ns) + (j * burst_gap);
-			ctx->nsec = start_ns + nsec;
-
-			start_param.tick_type = ODP_TIMER_TICK_ABS;
-			start_param.tick = start_tick + odp_timer_ns_to_tick(timer_pool, nsec);
-			start_param.tmo_ev = ctx->event;
-
-			ret = odp_timer_start(ctx->timer, &start_param);
+			if (mode == MODE_PERIODIC) {
+				odp_timer_periodic_start_t start_param;
+
+				nsec = offset_ns + (j * burst_gap);
+				ctx->nsec = start_ns + nsec;
```

Comment:
If nsec = 0 -> first_tick = 0, should one period be added to `ctx->nsec`?

Reply:
One period added in v5. I will post a separate PR to fix the timer implementation correspondingly.